src/schoology-extractor/schoology_extractor/grading_periods.py

```diff
@@ -3,14 +3,32 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
+import logging
 import os
+from typing import Any, List
+import sys
 
 from dotenv import load_dotenv
 
 from helpers import export_data
 from api.request_client import RequestClient
 
 load_dotenv()
+# Configure logger
+log_level = os.getenv("SCHOOLOGY_LOG_LEVEL", "INFO")
+assert log_level in ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"], "The specified `SCHOOLOGY_LOG_LEVEL` is not valid"
+logFormatter = '%(asctime)s - %(levelname)s - %(message)s'
+
+logging.basicConfig(
+    handlers=[
+        logging.StreamHandler(sys.stdout),
+    ],
+    format=logFormatter,
+    level=log_level
+)
+logger = logging.getLogger(__name__)
+
+logger.debug("Loading and processing .env variables")
```

Comment:
We're processing environment variables in general, which _might_ have been setup with dotenv.

```suggestion
logger.debug("Loading and processing environment variables")
```